### ISSUE TITLE
Bug: Tensorflow version is incorrect for AutoKeras 2.0

### ISSUE THREAD

**Issue by spartanjoax** [keras-team/autokeras#1940]
### Bug Description

The Readme states that AutoKeras runs on Tensorflow 2.8 but when installing, a dependency is Keras 3.0 which requires at least Tensorflow 2.16

**Comment by haifeng-jin** [comment]
Thanks!
I have updated the README by removing the TF and Python label as they are constantly changing by the dependencies.
